dao/src/main/java/greencity/entity/order/CourierLocations.java

```diff
@@ -0,0 +1,47 @@
+package greencity.entity.order;
+
+import greencity.entity.enums.CourierLimit;
+import greencity.entity.user.Location;
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Builder
+@EqualsAndHashCode(exclude = {"courier", "orders", "location"})
+@Table(name = "courier_locations")
+public class CourierLocations {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "min_amount_of_big_bags")
+    private Long minAmountOfBigBags;
+
+    @Column(name = "max_amount_of_big_bags")
+    private Long maxAmountOfBigBags;
+
+    @Column(name = "min_price_of_order")
+    private Long minPriceOfOrder;
+
+    @Column(name = "max_price_of_order")
```

Comment:
I think name is not required as long column name can be resolved based on fieldName.
In this case `maxPriceOfOrder` will be resolved into column `max_price_of_order`
`
@Column
private Long maxPriceOfOrder;
`